addons/stock/views/stock_move_views.xml

```diff
@@ -183,7 +183,7 @@
                 <tree editable="bottom" decoration-muted="state == 'done' and is_locked == True" decoration-success="product_uom_qty==qty_done" decoration-danger="qty_done &gt; product_uom_qty and state != 'done'">
                     <field name="picking_id" invisible="1"/>
                     <field name="product_id" invisible="1"/>
-                    <field name="location_id" invisible="not context.get('show_source_location')" domain="[('id', 'child_of', parent.location_id)])" groups="stock.group_stock_multi_locations"/>
+                    <field name="location_id" invisible="not context.get('show_source_location')" groups="stock.group_stock_multi_locations"/>
```

Comment:
What should we do? Do we need to duplicate twice this view and add them in two inline views instead of using tree_view_ref? Or create a related field parent_location_id on stock_move_line?

Reply:
@amoyaux We can also add the missing field `location_id` in the parent view(s).

Reply:
I added `location_id` in the superview.

Reply:
Actually this domain had a typo (extra closing parenthesis).
So I just fixed the typo :smile: